Makefile

```diff
@@ -9,14 +9,16 @@ COPY_DEREFERENCED=cp -RH
 RM_FILE=rm -f
 RM_DIR=rmdir -p --ignore-fail-on-non-empty
 
-ETCDIR_NX   ?= /etc/nxagent
-PREFIX      ?= /usr/local
-BINDIR      ?= $(PREFIX)/bin
-LIBDIR      ?= $(PREFIX)/lib
-USRLIBDIR   ?= $(LIBDIR)
-INCLUDEDIR  ?= $(PREFIX)/include
-NXLIBDIR    ?= $(LIBDIR)/nx
-CONFIGURE   ?= ./configure
+ETCDIR_NX       ?= /etc/nxagent
+PREFIX          ?= /usr/local
+BINDIR          ?= $(PREFIX)/bin
+LIBDIR          ?= $(PREFIX)/lib
+SHLIBDIR        ?= $(LIBDIR)
+NXLIBDIR        ?= $(SHLIBDIR)/nx
+USRLIBDIR       ?= $(NXLIBDIR)/X11
+USRLIBDIRPATH   ?= $(USRLIBDIR)
```

Comment:
USRLIBDIRPATH override has been removed now. The update branch has been rebased against latest 3.6.x. Please merge then.